types/read-pkg-up/read-pkg-up-tests.ts

```diff
@@ -0,0 +1,4 @@
+import * as ReadPkgUp from 'read-pkg-up';
```

Comment:
Please use `import ReadPkgUp from require('read-pkg-up');` as discussed in [module-function.d.ts](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html).